department-of-veterans-affairs/caseflow
# List selected filter options in aria label of filter column icon

Resolves #12134

### Description
Ensures screen readers will read what a column is filtered by

### Acceptance Criteria
- [ ] When no filters are selected only the column name is read
- [ ] When filters are applied, "Column_name filtered by filter_criteria" is read

### Testing Plan
1. Go to any queue with multiple filter options, I chose BVAAAAAAAAABSHIRE's assign queue http://localhost:3000/queue/3/assign
1. Use voice over to navigate to the Task(s) column filter Icon
1. Ensure this reads "Filter by type"
1. Apply "Original" filter
1. Use voice over to navigate back to the Task(s) column filter Icon
1. Ensure this reads "Filter by type. Filtered by original"
1. Apply "Post Remand" filter as well
1. Use voice over to navigate back to the Task(s) column filter Icon
1. Ensure this reads "Filter by type. Filtered by original, post remand"
